plugins/bulk_upload/drop_folder_xml/lib/kDropFolderXmlEventsConsumer.php

```diff
@@ -190,20 +190,9 @@ private function setFileError(DropFolderFile $file, $status, $errorCode, $errorD
 	 */
 	private function onXmlDropFolderFileStatusChangedToPurged(DropFolder $folder, DropFolderFile $file)
 	{
-		if($this->isXmlFile($file->getFileName(), $folder))
-		{
-			$statuses = array(DropFolderFileStatus::PARSED);
-			$parsedDropFolderFiles = DropFolderFilePeer::retrieveByLeadIdAndStatuses($file->getLeadDropFolderFileId(), $statuses);
-			if($parsedDropFolderFiles)
-			{
-				foreach ($parsedDropFolderFiles as $parsedFile) 
-				{
-					$parsedFile->setStatus(DropFolderFileStatus::PURGED);
-					$parsedFile->save();
-				}
-			}
-		}
 		
+		$xmlFileHandler = kDropFolderXmlFileHandler::getHandlerInstance($folder->getType());
```

Comment:
@tan-tan-kanarek  I didn't think it was correct to add a new event consumer for just 1 new drop folder type. I can change this if required.